idx/memory/memory.go

```diff
@@ -706,6 +706,9 @@ func (m *UnpartitionedMemoryIdx) Get(id schema.MKey) (idx.Archive, bool) {
 	m.RLockHigh()
 	defer m.RUnlockHigh()
 	def, ok := m.defById[id]
+	if !ok {
```

Comment:
writeQueue is optional, so we should check if it exists to avoid panic.
```suggestion
	if !ok && m.writeQueue != nil {
```

Reply:
Doesn't look resolved...

Reply:
Committed without add... Sorry